pkg/gui/custom_commands.go

```diff
@@ -136,7 +136,12 @@ func (gui *Gui) GenerateMenuCandidates(commandOutput, filter, valueFormat, label
 		return nil, gui.surfaceError(errors.New("unable to parse value format, error: " + err.Error()))
 	}
 
-	descTemp, err := template.New("format").Parse(labelFormat)
+	colorFuncMap := template.FuncMap{}
+	for k, v := range style.ColorMap {
+		colorFuncMap[k] = v.Foreground.Sprint
+	}
```

Comment:
Maybe it's better to abstract this logic into `pkg/gui/style/basic_styles.go`

In `basic_styles.go`:
```go
func TemplateFuncMapAddColors(m template.FuncMap) template.FuncMap {
	for k, v := range ColorMap {
		m[k] = v.Foreground.Sprint
	}
	m["underline"] = color.OpUnderscore.Sprint
	m["bold"] = color.OpBold.Sprint
	return m
}
```

And then in this file:
```suggestion
	colorFuncMap := style.TemplateFuncMapAddColors(template.FuncMap{})
```

Reply:
Sure, this way it'll be easier to use and we have to maintain only one list of colors. Thanks!